src/lando/utils/phabricator.py

```diff
@@ -212,6 +212,21 @@ def call_conduit(self, method: str, **kwargs) -> Any:  # noqa: ANN401
         PhabricatorAPIException.raise_if_error(response)
         return response.get("result")
 
+    def get_conduit_result_data(self, method: str, **kwargs) -> dict:
```

Comment:
Type hint could be more specific here: `dict[str, list[Any]]:`

Bikeshed: `call_conduit_paginated()` would make it clearer what this method is doing. A more detailed docstring would also be nice.

Reply:
nit: I think "collated" is more appropriate here, since we're "de-paginating" results rather than paginating them.